.editorconfig

```diff
@@ -0,0 +1,200 @@
+# editorconfig.org
```

Comment:
Having a .editorconfig for the samples isn't necessary a bad idea, but it's a good idea to split this off to another PR. Specifically, there are two different .editorconfig files in this PR, which also aren't the same as the one we currently use in EF Core itself.

Reply:
I noticed that my CodeCleanup had wobbled some of the formatting, so I added mine as an explicit .editorconfig rather than have ambiguity. Got carried away and added 2 [whoops, like NYC so good they named it twice]. Happy that you should ditch my templates in favour of an "official one". I liked the simplicity of Roji's original that seemed to command instant wisdom of just pulling what you actually need (just cols/rows critters required not the whole prairie], but spotting the integer-division just had to follow-thru with a full-on OCD response! Honestly, not trying to over-complicate Roji's simple v1 main-dish, but to offer some vn+1 dessert as next course [maybe this should appear later once the newbie has joined the party and not frightened by too much noise. Yeah, maybe.

Anyway I have been exploring various Find/First style as well as Cold/Hot caching and even few basic ADO flavours for comparison & horses/courses. Missing so far is a commentary of when/what suitable, but I hope the BM results will speak for themselves. Your call, and I hope to revise PR 2965 with much more meat shortly and hope doesn't give you too much indigestion as pure vegans. [ok I give up speech-writing and will stay with my day-job!].

Thanks for your launching this perf journey as I find on my consultancy journey that there are many "unknown unknowns" that need a strong torch.